omod-1.9/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs1_9/ObsResource1_9.java

```diff
@@ -42,12 +45,31 @@ public static void setConcept(Obs obs, Object value) {
 				identifier = uuid;
 			}
 		}
-		
+
 		if (identifier == null) {
 			identifier = value;
 		}
-		
+
 		obs.setConcept(ConversionUtil.getConverter(Concept.class).getByUniqueId((String) identifier));
 	}
-	
+
+
+    @Override
+    protected SimpleObject convertDelegateToRepresentation(Obs delegate, DelegatingResourceDescription rep) {
+        try {
+            return super.convertDelegateToRepresentation(delegate, rep);
+        } catch (ConversionException e) {
```

Comment:
Is this the lowest down we can get?  Seems like there might be other cases where a conversion exception is thrown and these conditions are true (though relatively unlikely, is possible), and this would end up swallowing those.

Reply:
This was as low as I think I could get without hacking something into the shared superclass, which I didn't want to do.

Note that if the Obs does *not* have a value drug or a comment indicating it's a value location, it just re-throws the error.  And if it *is* one of those two, it just runs the conversion again removing the "value" property, without trapping an other error that may occur.  So it seemed rather unlikely.

I did also want to set it so it so it only trapped the error if it was a Custom Rep that was being requested, but that would need to have happened higher up and left me with less control with what I could actually return.